Mathlib/Tactic/Basic.lean

```diff
@@ -9,6 +9,8 @@ import Mathlib.Tactic.ExtendDoc
 import Mathlib.Tactic.Lemma
 import Mathlib.Tactic.TypeStar
 import Mathlib.Tactic.Linter.OldObtain
+-- To test that the new simproc doesn't break anything
+import Mathlib.Tactic.Simproc.ExistsAndEq
```

Comment:
I think this might make sense to import near wherever the non-simproc lemmas about `Exists` and `And` are; but maybe @kim-em has opinions.